docs/handler-pan.md

```diff
@@ -47,45 +47,84 @@ See [set of properties inherited from base handler class](handler-common.md#prop
 Minimum distance the finger (or multiple finger) need to travel before the handler [activates](state.md#active). Expressed in points.
 
 ---
-### `minOffsetX`
+### `minPointers`
 
-Minimum distance expressed in points along X axis the finger (or multiple finger) need to travel before the handler [activates](state.md#active). If set to a negative value we expect the finger to travel "left" by the given distance. When set to a positive number the handler will activate on a movement to the "right". If you wish for the movement direction to be ignored use [`minDeltaX`](#mindeltax) instead.
+A number of fingers that is required to be placed before handler can [activate](state.md#active). Should be a positive integer.
 
 ---
-### `minOffsetY`
+### `maxPointers`
 
-Minimum distance expressed in points along Y axis the finger (or multiple finger) need to travel before the handler [activates](state.md#active). If set to a negative value we expect the finger to travel "up" by the given distance. When set to a positive number the handler will activate on a movement to the "bottom". If you wish for the movement direction to be ignored use [`minDeltaY`](#mindeltay) instead.
+When the given number of fingers is placed on the screen and handler hasn't yet [activated](state.md#active) it will fail recognizing the gesture. Should be a positive integer.
 
 ---
-### `minDeltaX`
+### `activeOffsetX`
 
-Minimum distance expressed in points along X axis the finger (or multiple finger) need to travel (left or right) before the handler [activates](state.md#active). Unlike [`minoffsetx`](#minoffsetx) this parameter accepts only non-negative numbers that represents the distance in point units. If you want for the handler to [activate](state.md#active) for the movement in one particular direction use [`minOffsetX`](#minoffsetx) instead.
+Range along X axis where fingers travels without activation of handler. Moving outside this range implies activation of handler. Range could be given as an array or a single number.
+ If range is set as array, first value must be negative, a the second one positive.
```

Comment:
as an array

negative -> lower or equal to 0
positive -> higher or equal to 0